app/ui/src/app/customizations/api-connector/api-connector-list.component.ts

```diff
@@ -24,16 +26,19 @@ const category = getCategory('ApiConnectors');
   styleUrls: ['./api-connector-list.component.scss']
 })
 export class ApiConnectorListComponent implements OnInit {
-  connectors: Observable<Connectors>;
-  filteredConnectors: Subject<Connectors> = new BehaviorSubject(<Connectors>{});
-  loading: Observable<boolean>;
+  apiConnectors$: Observable<ApiConnectors>;
+  filteredApiConnectors$: Subject<ApiConnectors> = new BehaviorSubject(<ApiConnectors>{});
```

Comment:
You do not need to annotate class members when initializing those straight away. The TypeScript compiler will do that by type inference:

```typescript
filteredApiConnectors$ = new BehaviorSubject(<ApiConnectors>{});
``` 

will suffice. Also, let's keep consistency between types. `BehaviorSubject` is an specialized class that inherits from `Subject`. If you want to use `BehaviorSubject`, you should annotate the class member accordingly (although not required in this case)